modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/endpoint/SchedulerRestService.java

```diff
@@ -1793,4 +1852,62 @@ private Properties parseProperties(String serializedProperties) throws IOExcepti
     caProperties.load(new StringReader(serializedProperties));
     return caProperties;
   }
+
+  /**
+   * Serializes mediapackage schedule metadata into JSON array string.
+   *
+   *{
+   *   "events": [
+   *    {
+   *        "metadata": "",
+   *        "attachments": "",
+   *        "id": "mp1",
+   *        "media": "",
+   *        "publications": ""
+   *     },
+   *    {
+   *        "metadata": "",
+   *        "attachments": "",
+   *        "id": "mp2",
+   *         "media": "",
+   *         "publications": ""
+   *     }
+   * ],
+   *"totalCount": "2"
+   *}
+   *
+   * @return serialized array as json array string
+   * @throws SchedulerException 
+   * @throws ParseException
+   */
+  @SuppressWarnings("unchecked")
```

Comment:
The org.json.simple.JSONObject and org.json.simple.JSONOArray have unchecked warnings with put and add because they don't support generics. I removed the @SuppressWarnings("unchecked") so that it is obvious where the warnings are.